package/root/usr/local/sbin/rock64_eth0_stats.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+set -e
+
+if [ "$(id -u)" -ne "0" ]; then
```

Comment:
In this case, nothing, it was a leftover of the original health script. Permissions in this case allow 'everyone' to read.